compute_tools/src/extension_server.rs

```diff
@@ -0,0 +1,244 @@
+// Download extension files from the extension store
+// and put them in the right place in the postgres directory (share / lib)
+/*
+The layout of the S3 bucket is as follows:
+├── 111
+    │   ├── v14
+    │   │   ├── extensions
+    │   │   │   ├── anon.tar.zst
+    │   │   │   └── embedding.tar.zst
+    │   │   └── ext_index.json
+    │   └── v15
+    │       ├── extensions
+    │       │   ├── anon.tar.zst
+    │       │   └── embedding.tar.zst
+    │       └── ext_index.json
+    ├── 112
```

Comment:
I don't get this diagram.

Looks like it represents path `/111/112`. What does that mean?

Or is it just an unneeded tab in this line?

Reply:
the diagram was wrong. thanks for pointing this out!